src/main/java/com/libertymutualgroup/herman/aws/ecs/EcsPushDefinition.java

```diff
@@ -160,11 +193,13 @@ public void setTopics(List<SnsTopic> topics) {
         this.topics = topics;
     }
 
+    @Override
     public List<DynamoDBTable> getDynamoDBTables() {
         return dynamoDBTables;
     }
 
-    public void setDynamoDBTables(List<DynamoDBTable> dynamoDBTables) {
+    public void setDynamoDBTables(
+        List<DynamoDBTable> dynamoDBTables) {
```

Comment:
Not a big deal, but this formatting feels a little funny

Reply:
Ah, I usually just run auto-format in IJ and this is what it does based on line length. Maybe we should check in a code style scheme? At least agree to a hard wrap at X columns sort of deal?

Reply:
Yeah, that's probably a good idea!